### PR TITLE
cmds: Rename util 'sleep' to 'msleep'

### PR DESCRIPTION
**Pull request by anton-bondarev**

### CODE REVIEW

File: platform/stm32f4_cnc/http_admin/cgi-bin/live [hunk 1]
```diff
@@ -11,5 +11,5 @@ id=0
 while true; do
 	echo -ne "data: { \"id\" : $id }\n\n" 2>/dev/null || exit 0
 	id=$(($id + 1))
-	sleep 1
```
Comment (AntonKozlov) [inline platform/stm32f4_cnc/http_admin/cgi-bin/live]: the script runs on host, should not be changed

File: platform/stm32f4_cnc/http_admin/cgi-bin/live_status [hunk 2]
```diff
@@ -11,5 +11,5 @@ id=0
 while true; do
 	echo -ne "data: { \"id\" : $id }\n\n" 2>/dev/null || exit 0
 	id=$(($id + 1))
-	sleep 1
```
Comment (AntonKozlov) [inline platform/stm32f4_cnc/http_admin/cgi-bin/live_status]: same

File: src/cmds/msleep/Mybuild [hunk 3]
```diff
@@ -2,21 +2,21 @@
...
 		DESCRIPTION
 
 		AUTHOR
 			Anton Kozlov
 	''')
-module sleep {
+module msleep {
 	@IncludePath("$(SRC_DIR)/compat/posix/include")
-	source "sleep.c"
+	source "msleep.c"
```
Comment (AntonKozlov) [inline src/cmds/msleep/Mybuild]: msleep.c have usage(), that still have "USEC"

Comment (anton-bondarev) [reply src/cmds/msleep/Mybuild]: @AntonKozlov Thanks. Fixed it. Look up again, please.

**Pull request merged**